src/Material/Footer.elm

```diff
@@ -0,0 +1,590 @@
+module Material.Footer
+  exposing
+    ( Type(..)
+    , Property
+    , Content(..)
```

Comment:
Do we need to expose the constructors? If we don't, we can later change the internal representation (if we like) without inducing a major version bump.